e2e/test/TelemetryPropertyNames.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    /// <summary>
+    /// The property names to log to telemetry.
+    /// </summary>
+    public class TelemetryPropertyNames
```

Comment:
I would rename this to TestLoggingPropertyNames or something, something that doesn't use the word "telemetry".
"Telemetry" kind-of has an established meaning in the scope of Azure IoT Hub.

Reply:
Application Insights calls it telemetry and hence I named it that. It is not a customer facing API so I'm sort of on the fence. This is test telemetry just like IotHub telemetry. Let me think about it.

Reply:
True, I meant it might be confusing for us. When I first looked at the class, I was confused as to what it stood for.
How about `ApplicationInsightsTelemetryPropertyNames`?

Reply:
We can always use different telemetry providers so a generic name is better. Ex: We can log to AI. Kusto etc... We can always change implementation but we should not be changing class names. Maybe LoggerPropertyNames/LoggingPropertyNames?

Reply:
Yup, like that too: LoggingPropertyNames